app/scripts/views/complete_reset_password.js

```diff
@@ -169,7 +169,7 @@ define(function (require, exports, module) {
     PasswordMixin,
     PasswordResetMixin,
     PasswordStrengthMixin,
-    ResendMixin,
+    ResendMixin(),
```

Comment:
Is there a case for giving `ResendMixin` a slightly different name to the other mixins, because of the different usage pattern?